index.js

```diff
@@ -28,7 +28,9 @@ function isValidSchema (schema, name) {
       name = ''
     }
     const first = validate.errors[0]
-    const err = new Error(`${name}schema is invalid: data${first.dataPath} ${first.message}`)
+    const err = new Error(
+      `${name}schema is invalid: data${first.dataPath} ${first.message}`
+    )
```

Comment:
Can you avoid changing the format of the code?

Reply:
I will try, but this formatted with `standard` code-style. I manually didn't touched this part

Reply:
I'm formatted with `npx standard --fix`